inbox/mailsync/backends/imap/generic.py

```diff
@@ -343,7 +345,11 @@ def initial_sync(self):
     @retry_crispin
     def poll(self):
         log.bind(state='poll')
-        log.debug('polling')
+        # Only log every 5 minutes to cut down on the volume of
+        # this log statement
+        if time.time() - self.polling_logged_at > 60 * 5:
```

Comment:
Minor: Could save a call by doing `timestamp = time.time()` and using it in both places.

Reply:
I realized that after looking at the PR, you guilted me into fixing it.